### PR TITLE
Portfolio site 1

### PR DESCRIPTION
**Pull request by Becky449**
#Updated Portfolio site

- Added content on html file
- added styling to stylesheet
- Added background image
- Added social media icons image

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Becky449, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing linter checks.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a readme file that is not descriptive , kindly use [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) to create your readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1 +1,31 @@
...
+    <header class="nav-bar">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav>
+            <button type="button" class="hamburger">&#9776;</button>
+        </nav>
+    </header>
+    <main>
+        <section class="main-content">
+            <div class="content">
+                <h1>I'm Rabecca Glad to see you!</h1>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please increase the font weight for the main heading and make sure it matches the Figma design.

### Your design

image

### Figma design

image

File: index.html [hunk 2]
```diff
@@ -1 +1,31 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="nav-bar">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav>
+            <button type="button" class="hamburger">&#9776;</button>
+        </nav>
```
Comment (thecodechaser) [inline index.html:16]: - [ ] Please move the hamburger button(Icon) to a little right to match the Figma design.

### Your design

image

### Figma design

image

File: index.html [hunk 3]
```diff
@@ -1 +1,31 @@
...
+    </header>
+    <main>
+        <section class="main-content">
+            <div class="content">
+                <h1>I'm Rabecca Glad to see you!</h1>
+                <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                <a href="#" class="social">LET'S CONNECT</a><br>
+               <img src="images/social-icons.png" class="social-media-icons" alt="social-media-icons">
```
Comment (thecodechaser) [inline index.html]: - [ ] Please remove this image and use social media links with anchor tags. Also, remember to wrap your links in a list, When we have similar elements It is a good practice to wrap them in a list.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Becky449👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -1 +1,37 @@
...
+                <h1 class="content-text">I'm Rabecca Glad to see you!</h1>
+                <p class="content-text-p">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                <a href="#" class="social-connect">LET'S CONNECT</a>
+                <ul class="social-list">
+                    <li class="social-media-icons"><a href="https://twitter.com"><img src="twitter.png" alt=""></a></li>
+                    <li class="social-media-icons"><a href="https://www.linkedin.com"><img src="linkedin.png" alt=""></a></li>
+                    <li class="social-media-icons"><a href=""><img src="median.png" alt=""></a></li>
+                    <li class="social-media-icons"><a href="https://github.com"><img src="git.png" alt=""></a></li>
+                    <li class="social-media-icons"><a href="https://chilldigital.com.au"><img src="chill.png" alt=""></a></li>
+                </ul>
```
Comment (thecodechaser) [inline index.html]: - [ ] The images you are using here for social links are not available in your project. Please import all images and make sure they are visible and follow the Figma design.
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Becky449👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Becky449  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**